scuba/__main__.py

```diff
@@ -316,8 +319,8 @@ def __setup_native_run(self):
 
         # The user command is executed via a generated shell script
         with self.open_scubadir_file('command.sh', 'wt') as f:
-            self.docker_cmd += ['/bin/sh', f.container_path]
-            writeln(f, '#!/bin/sh')
+            self.docker_cmd += [context.shell, f.container_path]
+            writeln(f, '#!{}'.format(context.shell))
```

Comment:
We probably don't really need the she-bang line at all, since we're explicitly invoking the shell (via `self.docker_cmd`).

```suggestion
```

Reply:
Yeah, I agree. We'll have to re-work the tests, as they check the shebang line to validate that the shell was set correctly. I went that route because scuba's shell escaping prevents me from just running `scuba echo \$SHELL`, as `$SHELL` is escaped to `'$SHELL'` before being passed into the container. However, since the shebang isn't actually relevant here, it's probably best to change how we're validating the shell.

Reply:
I'm fine leaving it like this for now, unless you have an easy change in mind.

Reply:
I added 73c9a69 to address this conversation. I think the new way of checking the shell is more correct/rigorous.